test/controllers/api/v2/fact_values_controller_test.rb

```diff
@@ -11,23 +11,23 @@ def setup
     get :index
     assert_response :success
     fact_values = ActiveSupport::JSON.decode(@response.body)['results']
-    expected_hash = {@host.name=>{"kernelversion"=>"2.6.9"}}
+    expected_hash = {@host.name => {"kernelversion" => "2.6.9"}}
```

Comment:
are changes in this file related? (I like them :)

Reply:
@ares that's the first of the two commits.